lib/proxy/plugin_initializer.rb

```diff
@@ -114,7 +114,7 @@ def configure
   end
 
   def fail_group(an_exception)
-    fail_group_with_message("Disabling all modules in the group #{printable_module_names(member_names)} due to a failure in one of them: #{an_exception}", an_exception.backtrace)
+    fail_group_with_message("Disabling all modules in the group #{printable_module_names(member_names)} due to a failure in one of them: #{an_exception}", an_exception)
```

Comment:
When debugging this, backtrace was not printed correctly. It's because the method accepts exception, not backtrace array.